lib/services/storage_service.dart

```diff
@@ -0,0 +1,47 @@
+import 'package:shared_preferences/shared_preferences.dart';
+import 'dart:async';
+
+class StorageUtil {
+  static StorageUtil _storageUtil;
+  static SharedPreferences _preferences;
+
+  static Future<StorageUtil> getInstance() async {
+    if (_storageUtil == null) {
+      // keep local instance till it is fully initialized.
+      var secureStorage = StorageUtil._();
+      await secureStorage._init();
+      _storageUtil = secureStorage;
+    }
+    return _storageUtil;
+  }
+
+  StorageUtil._();
+
+  Future _init() async {
+    _preferences = await SharedPreferences.getInstance();
+  }
+
+  // get string
+  static String getString(String key, {String defValue = ''}) {
```

Comment:
no.  the store service won't be like this this. i.e getString putString. etc. this is not of use.

the function need to be like loginUser  logoutUser etc
i.e the specific case we are using storage for

Reply:
can u take action on this

Reply:
it's pending still, we see to modify it

Reply:
@manishiitg modified it

Reply:
this is not done. discuss with me